### PR TITLE
Naveen hw1

### PR DESCRIPTION
**Pull request by naveenpardeep**
please check mam.

Thank You
Yours Sincerely 
Naveen

### CODE REVIEW

**Comment by MOUBAREK1998** [comment]
thank you

File: Naveen-hw1/task1.c [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            {
+                // grandchild2
+                printf("Grandchild2 N4 and pid is %d parent pid %d \n", getpid() , getppid());
+               
+            }
+          
+           }
+    }
+    
+}
```
Comment (NataRazmochaeva) [inline Naveen-hw1/task1.c:38]: Task 1 is accepted

File: Naveen-hw1/task2.c [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    pid_t parent , child1 , child2, grandchild;
+ 
+    printf("Parent process N1 and pid %d\n", getpid());
+
+  child1 = fork();
+        if (child1 == 0)
+        {
+            // child1
+            printf("child1 N2 and pid is %d\n", getpid());
+
```
Comment (NataRazmochaeva) [inline Naveen-hw1/task2.c:16]: Please add outputting of ppid

File: Naveen-hw1/task2.c [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        }
+       else if (child1 > 0)
+        {
+           
+            child2 = fork();
+            if (child2 == 0)
+            {
+                // child2
+                printf("child2 N3 and pid is %d\n", getpid());
+
```
Comment (NataRazmochaeva) [inline Naveen-hw1/task2.c:26]: Please add outputting of ppid

File: Naveen-hw1/task2.c [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+            if (child2 == 0)
+            {
+                // child2
+                printf("child2 N3 and pid is %d\n", getpid());
+               
+        
+            grandchild = fork();
+            if (grandchild == 0)
+            {
+                printf("N3 parent of N4 and pid is  %d\n",getppid());
```
Comment (NataRazmochaeva) [inline Naveen-hw1/task2.c:31]: Please add outputting of ppid

**Review by NataRazmochaeva — CHANGES_REQUESTED** [review]
Some remarks to task 2

**Pull request merged**